executor/executor_test.go

```diff
@@ -3378,9 +3378,9 @@ func (s *testSuite) TestSelectView(c *C) {
 	tk.MustExec("use test")
 	tk.MustExec("create table view_t (a int,b int)")
 	tk.MustExec("insert into view_t values(1,2)")
-	tk.MustExec("create view view1 as select * from view_t")
-	tk.MustExec("create view view2(c,d) as select * from view_t")
-	tk.MustExec("create view view3(c,d) as select a,b from view_t")
+	tk.MustExec("create definer='root'@'localhost' view view1 as select * from view_t")
```

Comment:
It's better to remain the old tests.

Reply:
Due to our test frame, old test without `definer='root'@'localhost'` will fall

Reply:
both test case is standard MySQL grammar.

Reply:
@XuHuaiyu Current test framework will not set `b.ctx.GetSessionVars().User` correctly.